modules/gitops-release-notes-1-9-1.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-9-1_{context}"]
+= Release notes for {gitops-title} 1.9.1
+
+{gitops-title} 1.9.1 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-9-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:3591- {gitops-title} 1.9.1 security update advisory 
+
+Issued: 2023-07-10
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:3591[RHSA-2023:3591]
```

Comment:
same comment about RHSA #

Reply:
To be updated once we have the required link